ChatApp/ChatView/MessageView.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  MessageView.swift
+//  ChatApp
+//
+//  Created by Giorgi Makadze on 19.04.2023.
+//
+
+import Foundation
+import UIKit
+
+class MessageView: UIView, UITextViewDelegate, UICollectionViewDelegate {
+    private let textView = UITextView() // texfield
+    
+    private let button = UIButton(type: .custom)
+    private var collectionView: UICollectionView!
+    private var dataSource: UICollectionViewDiffableDataSource<Section, Message>!
+    let conteinerView = UIView()
+    
+    struct Message: Hashable {
+        let text: String
+        let date: String
+        
+        init(text: String, date: String) {
+            self.text = text
+            self.date = date
+        }
+    }
+    
+    let dataItems = [
+        Message(text: "გამარჯობა, ზეზვა როგორ ხარ?", date: "მარ 14,16:00"),
+        Message(text: "გამარჯობა, მზია როგორ ხარ?", date: "მარ 14,16:05"),
+        Message(text: "კარგად!", date: "მარ 14,16:10"),
+        Message(text: "ნუ მატყუებ!", date: "მარ 14,16:11")
+        
+    ]
+    
+    enum Section {
+        case main
+    }
```

Comment:
are your going to add some other options in this enum? if you will only have one "main" case you do not need this enum.

Reply:
Yes, I plan to add it in the future